docs/man_pages/general/autocomplete-disable.md

```diff
@@ -16,7 +16,8 @@ Command | Description
 [autocomplete](autocomplete.html) | Asks for confirmation if command line autocompletion should be enabled for bash and zsh.
 [autocomplete-status](autocomplete-status.html) | Prints the current status of your command-line completion settings.
 [autocomplete-enable](autocomplete-enable.html) | Enables command-line completion for bash and zsh shells.
-[feature-usage-tracking](feature-usage-tracking.html) | Configures anonymous usage statistics tracking for the NativeScript command-line interface.
+[feature-usage-tracking](feature-usage-tracking.html) | Configures anonymous usage statistics tracking for the AppBuilder CLI.
+[exceptions-tracking](exceptions-tracking.html) | Configures anonymous exceptions tracking for the Telerik AppBuilder CLI.
 [help](help.html) | Lists the available commands or shows information about the selected command.
-[doctor](doctor.html) | Checks your system and reports potential problems which may prevent developing with AppBuilder CLI.
+[doctor](doctor.html) | Checks your system and reports potential problems which may prevent developing with Telerik AppBuilder CLI.
```

Comment:
Checks your system for configuration problems which might prevent the Telerik AppBuilder CLI from working properly.